src/parser.rs

```diff
@@ -72,16 +75,20 @@ named!(fname<String>, alt!(quoted | bare));
  * Header lines
  */
 
-named!(header_line_content<String>,
+named!(header_line_content<(String, Option<DateTime<FixedOffset>>)>,
     chain!(
         filename: fname ~
-        take_until!("\n") ,
+        opt!(space) ~
```

Comment:
Oh nice! didn't know space accepted tabs, that's great.

I guess my question for where `space` lives is about how strict we should be. My current expectation is that we'll see two kinds of header lines:

```
indicator ~ space ~ filename ~ newline
indicator ~ space ~ filename ~ space ~ date_or_whatever ~ newline
```

and that we'd never expect to see

```
indicator ~ space ~ filename ~ space ~ newline
indicator ~ space ~ filename ~ date_or_whatever ~ newline
```

If the `space` after filename depends on `after` being present, we accept the first two but reject the second two. If it is `opt!(space)` as you have it, we'd accept any of the four.

I'm happy to be less strict for now and go with the optional `space` in all cases :)